test/built-ins/Array/from/mapfn-returns-integer.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: mapFn returns a non-integer

```

Comment:
The filename (`mapfn-returns-integer.js`) does not match the description ("mapFn returns a non-integer"). Since the test body clearly concerns the behavior when an object is returned, probably it's just the filename that needs changing.